snippets/javascript/typescript.json

```diff
@@ -216,5 +216,38 @@
         "prefix": "#endregion",
         "body": ["//#endregion"],
         "description": "Folding Region End"
+    },
```

Comment:
What's the reasoning here? 

I'm not seeing much value added with the repetition on these snippets, you could reduce this PR to a single snippet: 

```json
"let-declaration": {
   "prefix": "let",
   "body": [ "let ${1:name}: ${2: type} = $3" ],
   "description": "declare a variable with type annotation"
 }
 ```
 
 Even then it is still a "clunky" because most people don't want snippets triggering after typing three characters (which you snippets do address by having the type as part of the `prefix`). 
 
Let me know your thoughts

Reply:
hahaha you're right I didn't think of that way, maybe I'll change it